.github/ISSUE_TEMPLATE/member_build_cache_issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 'Internal: Issue for the `build-cache` team'
+about: 'Report a problem about the Gradle build-cache'
+labels: '@build-cache, from:member'
+---
+
+<!--- TODO -->
+
+---
+cc: @gradle/build-cache
```

Comment:
This will make sure that we are all subscribed, right? I think I like it.

Reply:
Hope so, don't have a sandbox to validate it, but that's the idea, yes